src/containers/Article/Create/actions.test.js

```diff
@@ -0,0 +1,61 @@
+import configureMockStore from 'redux-mock-store';
+import fetchMock from 'fetch-mock';
+import thunk from 'redux-thunk';
+
+import { ARTICLE_SUCCESS, ARTICLE_FAILURE, ARTICLE_REQUEST } from './constants';
+import createArticleAction, { articleFailure, articleFetch, articleSuccess } from './actions';
+import getAction from '../../../utils/getActions';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('article actions', () => {
+  afterEach(() => {
+    fetchMock.reset();
+    fetchMock.restore();
+  });
+
+  it('should create an action to fetch article', () => {
+    const expectedAction = {
+      type: ARTICLE_REQUEST,
+    };
+    expect(articleFetch()).toEqual(expectedAction);
+  });
+
+  it('should create an action to dispatch errors on article fetch failture', () => {
+    const errors = {
+      errors: {
+        error: ['Article not Found'],
+      },
+    };
+    const expectedAction = {
+      type: ARTICLE_FAILURE,
+      errors,
+    };
+    expect(articleFailure(errors)).toEqual(expectedAction);
+  });
+
+  it('should create an action to dispatch success response.', () => {
+    const payload = {
+      user: {
```

Comment:
Let's have one file that contains an object with all our **test_data** to avoid duplication

Reply:
Yeah I agree. Our code should be as DRY as possible. Working on it